lib/lita/handlers/reviewme.rb

```diff
@@ -63,13 +70,24 @@ def generate_assignment(response)
       end
 
       def comment_on_github(response)
-        repo = response.matches.flatten.first
-        id = response.matches.flatten.last
+        repo = response.match_data[:repo]
+        id = response.match_data[:id]
         reviewer = next_reviewer
         comment = github_comment(reviewer)
 
-        github_client.add_comment(repo, id, comment)
-        response.reply("#{reviewer} should be on it...")
+        begin
+          github_client.add_comment(repo, id, comment)
+          response.reply("#{reviewer} should be on it...")
+        rescue
+          url = response.match_data[:url]
+          response.reply("I don't know what to do with that URL. #{chat_mention(reviewer, url)}")
```

Comment:
Since this is GH specific, should we reply with something like "Oops, maybe you need to grant the bot access to this repo"?